code-of-conduct/guide-code-of-conduct.md

```diff
@@ -0,0 +1,151 @@
+# Code of Conduct template guide
+
+---
+Thank you for downloading the Code of Conduct template from The Good Docs Project!
+We welcome any kind of feedback on our templates, both positive and negative.
+To suggest improvements, open an issue on our (https://github.com/thegooddocsproject/templates/issues)[Templates repository].
+---
+
+This is the guide that explains how to use the Good Docs Project Code of Conduct, which is part of the Code of Conduct template set.
+
+The Code of Conduct template set includes:
+
+- A `template-code-of conduct.md` file that you can use to create your community's public-facing Code of Conduct.
+- A `template-code-of-conduct-response-plan.md` file that you can use to create and explain the response policy and procedures your team will follow as you handle Code of Conduct incidents. This template is included in the same folder as the Code of Conduct template.
+- A `template-code-of-conduct-incident-record.md` file for a form that is filled out when a community moderator takes an incident report from a community members. This template is included in the same folder as the Code of Conduct template.
+- A `template-code-of-conduct-remediation-record.md` file for a form that is filled out when a community moderator meets with a community member to explain the consequences of a Code of Conduct violation. This template is included in the same folder as the Code of Conduct template.
+- The following guides:
+  - `guide-code-of-conduct.md`
+  - `guide-code-of-conduct-response-plan.md`
+  - `guide-code-of-conduct-incident-record.md`
+  - `guide-code-of-conduct-remediation-record`
+
+You might also consider using the `../our-team/template-our-team.md` template to let your community members know who they can contact to report a Code of Conduct violation. This document is not only necessary for Code of Conduct violations. It is a core document that helps you clearly communicate who belongs to your open source project or organization.
+
+
+## Why do I need a Code of Conduct and a response plan?
+
+A Code of Conduct is a lot like a good insurance policy: hopefully you won't have to use it because your community will always be healthy, productive, and inclusive.
+However, if or when a problem occurs in your community, you'll be glad that your Code of Conduct is there to support you.
+
+A Code of Conduct has many purposes and benefits. It can:
+
+- Help you define and clearly communicate your organization's mission, values, and guiding principles.
+- Encourage members of your community to behave ethically and inclusively.
+- Make your community a better place to collaborate and work.
+- Demonstrate to prospective community members that your community is one that is welcoming and safe to join.

```

Comment:
nitpick: Can we rephrase it as? Demonstrate to prospective community members that your community is warm, welcoming, and safe to join.

Reply:
Sure! I've adopted this suggestion.